state/src/storage/tx/mpc_preprocessing.rs

```diff
@@ -28,6 +32,26 @@ impl<'db, T: TransactionKind> StateTxn<'db, T> {
     ) -> Result<Option<PairwiseOfflineSetup>, StorageError> {
         self.inner().read(MPC_PREPROCESSING_TABLE, &preprocessing_key(cluster))
     }
+
+    /// Get the size of the preprocessing value for a cluster
+    pub fn get_mpc_prep_size(
+        &self,
+        cluster: &ClusterId,
+    ) -> Result<Option<PreprocessingSlice>, StorageError> {
+        let prep = res_some!(self.get_mpc_prep(cluster)?).values;
+        let slice = PreprocessingSlice {
+            num_bits: prep.random_bits.len(),
+            num_values: prep.random_values.len(),
+            num_input_masks: usize::min(
```

Comment:
it makes sense to use `usize::min` here regardless, but IIUC there should never be a case where the input masks have different lengths, right?

Reply:
that's correct